cl/people_db/import_judges/populate_fjc_judges.py

```diff
@@ -189,17 +187,17 @@ def add_positions_from_row(item, person, testing, fix_nums=None):
             position_type='jud',
             date_start=date_start,
             date_termination=date_termination,
-            court_id=courtid,
+            court_id=courtid
```

Comment:
We should always have trailing commas. They make adding/removing parameters in the future cleaner. Not all languages allow this, but Python, happily, does.

Reply:
I didn't know about trailing commas in Python and I'll slip these back in.  Very cool.

Reply:
Yeah, they're great. You went a little overboard though. You might find it useful to read Google's Python format guide: https://google.github.io/styleguide/pyguide.html

But the one thing I'll mention here is that you should have:

    some_function(arg1,
                  arg2)

Not:

    some_function(arg1,
                  arg2,)

I could probably drum up a reason why the former is preferred, but I think mostly it's just a style thing. No need to go back and fix this though...just mentioning it for the future.

Really, the above should be:

    some_function(arg1, arg2)

In any case!